atflow/trainers.py

```diff
@@ -101,6 +103,8 @@ def build(self):
 
             # configure to save and load all variables except for the global step value
             variables_to_load = [v for v in tf.global_variables() if 'global_step' not in v.name]
+            if self.skip_adam_variables:
+                variables_to_load = [v for v in variables_to_load if 'adam' not in v.name.lower()]
```

Comment:
Adam also stores a beta1_power and beta2_power variables:
I was also doing something like that manually:
```
all_but_adam = filter(lambda var: 'Adam' not in var.name, all_vars)
all_but_adam = filter(lambda var: 'beta1_power' not in var.name, all_but_adam)
all_but_adam = filter(lambda var: 'beta2_power' not in var.name, all_but_adam)
all_but_adam = filter(lambda var: 'global_step' not in var.name, all_but_adam)
```